pkg/runtime/bindings/registry.go

```diff
@@ -35,44 +35,44 @@ type Registry interface {
 	CreateInputBinding(name string) (bindings.InputBinding, error)
 }
 
-type bindingsRegistry struct {
+type BindingsRegistry struct {
 	outputBindingStores map[string]func() bindings.OutputBinding
 	inputBindingStores  map[string]func() bindings.InputBinding
 	info                *info.RuntimeInfo
 }
 
-func NewRegistry(info *info.RuntimeInfo) *bindingsRegistry {
+func NewRegistry(info *info.RuntimeInfo) *BindingsRegistry {
```

Comment:
There's no need to export `bindingsRegistry`. 
Instead, we can modify this constructor fuction to:
```go
func NewRegistry(info *info.RuntimeInfo) Registry {
```

Reply:
Good catch, this is a more elegant way!